File: Ad-Hoc/3205/3205.py
T = int(input())
for _ in range(T):
    r, e, c = [int(x) for x in input().split()]
    profitAdvertise = e - c
    
    ans = "does not matter"
    
    if profitAdvertise > r:     ans = "advertise"
    elif profitAdvertise < r:   ans = "do not advertise"
    print(ans)

File: Ad-Hoc/3205/3205.cpp
#include <iostream>
using namespace std;

int main() {
    int T;
    cin >> T;
    while(T--){
        int r, e, c;
        cin >> r >> e >> c;
        int profitAdvertise = e - c;

        if (profitAdvertise > r) cout << "advertise\n";
        else if (profitAdvertise < r) cout << "do not advertise\n";
        else cout << "does not matter\n";
    }
}